tests/integration/createAndEndActivities.intTest.ts

```diff
@@ -111,23 +111,25 @@ describe("PUT /activities/:id/end", () => {
         });
     });
 
-    context("when an existing activity is ended", () => {
-        it("should respond with HTTP 204", () => {
-            console.log(`Ending visit: ${postedActivity.id}`);
-            // End the just-create test activity
+    context("when an already ended activity is ended", () => {
```

Comment:
The context is same as the one after it. Doesn't clarify why `wasVisitAlreadyClosed=true` will be the response.

Reply:
you are right, nice catch thanks